mlvm/test_mlvm.py

```diff
@@ -0,0 +1,259 @@
+import os
+
+import pkg_resources
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+class MLVMTestCase(DataprocTestCase):
+    COMPONENT = "mlvm"
+    INIT_ACTIONS = ["mlvm/mlvm.sh"]
+    OPTIONAL_COMPONENTS = ["ANACONDA", "JUPYTER"]
+
+    def createCluster(self, configuration, **config):
+        config["optional_components"] = self.OPTIONAL_COMPONENTS
+        config["scopes"] = "cloud-platform"
+        config["timeout_in_minutes"] = 60
+        config["machine_type"] = "n1-standard-8"
+        
+        connector_metadata = "spark-bigquery-connector-version=0.13.1-beta"
+        metadata = config.get("metadata", "")
+        if metadata:
+            if "connector" not in metadata:
+                metadata = "{},{}".format(metadata, connector_metadata)
+        else:
+            metadata = connector_metadata
+        
+        config["metadata"] = metadata
+
+        super().createCluster(configuration, self.INIT_ACTIONS,
+                              **config)
+
+class ConnectorsTestCase(MLVMTestCase):
```

Comment:
May we just that some BQ job on a public dataset works, instead of re-testing connectors init action?